_pages/neurips_2019/neurips-2019-presenters.html

```diff
@@ -0,0 +1,11 @@
+---
+layout: main.nips
+permalink: /neurips-2019-presenters
+title: LXAI at NeurIPS
+published: true
```

Comment:
Please change published to false while you are waiting for the presenters' info so the site doesn't crash due to missing information.